### PR TITLE
SCALRCORE-16990 IACP API > Replace ingress-attributes with vcs-revisions 

### PR DESCRIPTION
**Pull request by penja**

### CODE REVIEW

**Review by mermoldy — APPROVED** [review]

File: vcs_revision.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	ID             string `jsonapi:"primary,vcs-revisions"`
+	Branch         string `jsonapi:"attr,branch"`
+	CommitSha      string `jsonapi:"attr,commit-sha"`
+	CommitMessage  string `jsonapi:"attr,commit-message"`
+	SenderUsername string `jsonapi:"attr,sender-username"`
+}
+
+// Read a workspace by its name.
+func (s *vcs_revisions) Read(ctx context.Context, vcsRevisionId string) (*VcsRevision, error) {
+	options := struct {
```
Comment (penja) [inline vcs_revision.go:34]: is typo thx @lbn

Comment (lbn) [inline vcs_revision.go:34]: No includable in this resource.

File: vcs_revision.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+var _ VcsRevisions = (*vcs_revisions)(nil)
+
+// workspaces implements Workspaces.
+type vcs_revisions struct {
+	client *Client
+}
+
+// VcsRevisions describes all the vcs revisions related methods that the Scalr API supports.
+type VcsRevisions interface {
+	// Read reads a vcs revisions by its ID.
```
Comment (lbn) [inline vcs_revision.go:19]: a VCS revision

File: vcs_revision.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import (
+	"context"
+	"fmt"
+	"net/url"
+)
+
+// Compile-time proof of interface implementation.
+var _ VcsRevisions = (*vcs_revisions)(nil)
+
+// workspaces implements Workspaces.
```
Comment (lbn) [inline vcs_revision.go:12]: Update please.

File: vcs_revision.go [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+// VcsRevision represents the VCS metadata
+type VcsRevision struct {
+	ID             string `jsonapi:"primary,vcs-revisions"`
+	Branch         string `jsonapi:"attr,branch"`
+	CommitSha      string `jsonapi:"attr,commit-sha"`
+	CommitMessage  string `jsonapi:"attr,commit-message"`
+	SenderUsername string `jsonapi:"attr,sender-username"`
+}
+
+// Read a workspace by its name.
```
Comment (lbn) [inline vcs_revision.go:32]: Update.

**Review by lbn — CHANGES_REQUESTED** [review]

**Review by lbn — APPROVED** [review]

**Pull request merged**